plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -42,6 +41,8 @@ import (
 	"k8s.io/kubernetes/plugin/pkg/scheduler/schedulercache"
 	schedutil "k8s.io/kubernetes/plugin/pkg/scheduler/util"
 	"k8s.io/metrics/pkg/client/clientset_generated/clientset"
+
+	"github.com/golang/glog"
```

Comment:
The order in my mind is "system lab", "k8s lab" and "3rd part lab" (split by empty line).